### ISSUE TITLE
Abount feature extraction on THUMOS14?

### ISSUE THREAD

**Issue by coder-LeeTao** [wzmsltw/BSN-boundary-sensitive-network#12]
Thank you for your great help!
I have another question. About THUMOS14, may I ask what is the way you take to extract features?
The FC layer output of anet is 200 classes. Did you modify the FC layer and then train on THUMOS14?

**Comment by wzmsltw** [comment]
Hi @coder-LeeTao , we did not finetune this network on THUMOS14. We direct use the output of last fc layer (without softmax) as the feature. Actually, you can also use the 2048-dim fc layer.
